exporter/f5cloudexporter/config.go

```diff
@@ -31,7 +31,7 @@ type Config struct {
 	Source string `mapstructure:"source"`
 
 	// AuthConfig represents the F5 Cloud authentication configuration options.
-	AuthConfig AuthConfig `mapstructure:"auth"`
+	AuthConfig AuthConfig `mapstructure:"f5cloud_auth"`
```

Comment:
This is a breaking change, why is this needed? Can this be avoided?

Reply:
That is correct, but it can't be avoided with the implementation that was chosen for Client Auth Extension, since it squashes into a he configured helpers. I have already communicated to users of it and internal F5 teams that this will break it and that we will be deprecating this in favor of an F5 Cloud Auth Extension + gRPC OTLP.

Reply:
Add an entry to the changelog then to explain it and motivation of it.